pods/spain-population-chart/spain-population-chart.config.ts

```diff
@@ -0,0 +1,42 @@
+export const svgDimensions = {
+  width: 1024,
+  height: 800,
+};
+
+export const svgBackgroundColor = "#FBFAF0";
+
+export const mapProjectionProps = {
+  scale: 3300,
+  translation: [500, 400],
+};
+
+export const provinceStyle = {
+  strokeWidth: 1,
+  stroke: "#2f4858",
+  fill: "#008c86",
+};
+
+export const selectedProvinceStyle = {
+  strokeWidth: 2,
+  stroke: "#bc5b40",
+  fill: "#f88f70",
+};
```

Comment:
We can make use of classes, check the approach that is already in master if we rename the css file to .module.css we get automatically support of css modules

Reply:
Something like this (?): 
![image](https://user-images.githubusercontent.com/33926302/85568805-4924f400-b632-11ea-8286-9f9abbd993ae.png)